src/sif/motorcyclecost.cc

```diff
@@ -493,7 +494,7 @@ bool MotorcycleCost::Allowed(const baldr::NodeInfo* node) const {
   return (node->access() & kMotorcycleAccess);
 }
 
-Cost MotorcycleCost::EdgeCost(const baldr::DirectedEdge* edge) const {
+Cost MotorcycleCost::EdgeCost(const baldr::DirectedEdge* edge, const float speed) const {
```

Comment:
did you override the EdgeCost methods in motorscooter, truck, pedestrian, etc. as well?